File: app/src/main/java/com/example/aichat/MyAssistant.java
package com.example.aichat;

public class MyAssistant {
    public String id;
    public String name;
    public String prompt;
    public String avatar;
    public String avatarImageBase64;
    public String type; // default / writer
    public boolean allowAutoLife;
    public boolean allowProactiveMessage;
    public SessionChatOptions options;
    public long updatedAt;

    public MyAssistant() {}
}

File: app/src/main/java/com/example/aichat/MyAssistant.kt
package com.example.aichat

data class MyAssistant(
    @JvmField var id: String = "",
    @JvmField var name: String = "",
    @JvmField var prompt: String = "",
    @JvmField var avatar: String = "",
    @JvmField var avatarImageBase64: String = "",
    @JvmField var firstDialogue: String = "",
    @JvmField var type: String = "", // default / writer
    @JvmField var allowAutoLife: Boolean = false,
    @JvmField var allowProactiveMessage: Boolean = false,
    @JvmField var options: SessionChatOptions? = null,
    @JvmField var updatedAt: Long = 0L
)
